modules/seqexec/web/client/src/main/scala/seqexec/web/client/components/sequence/steps/StepsTable.scala

```diff
@@ -641,19 +645,19 @@ object StepsTable extends Columns {
       b.props.rowGetter(i),
       b.props.canSetBreakpoint,
       b.state.breakpointHover) match {
-      case (-1, _, _, _) =>
+      case (-1, _, _, _)                                           =>
```

Comment:
Now that I look at this, the -1 is not that good. Should we give it a name, like `HeaderRow`?

Reply:
Ok, I'll look into it.

Reply:
The -1 is a virtualized thing. The `rowClassName` function where that code appears is passed directly to the virtualized component (which passes -1 to it when querying for the header class name). I'm defining a `HeaderRow` constant anyway for comparison.

Reply:
Right, it comes from virtualized. I just thought it would be better if we define it as our constant
If you want to do it on the facade please send a PR
It should probably be done in scala style: `HeaderRow`

Reply:
Even if it's a constant? (I defined it right below `MIDDLE_CLICK` ;) )